vm/src/obj/objframe.rs

```diff
@@ -24,6 +24,15 @@ impl FrameRef {
         "<frame object at .. >".to_owned()
     }
 
+    #[pymethod(name = "__delattr__")]
+    fn delattr(self, value: PyObjectRef, vm: &VirtualMachine) {
+        println!("Called: FrameRef::f_trace over {:?}", value);
```

Comment:
nice workaround for now. putting an incompatibility comment will be helpful for future work.